app/views/help.mustache

```diff
@@ -5,35 +5,47 @@
   <h2 id="search">Search</h2>
   <p>We support the following search expressions:</p>
   <ul>
-    <li><code>"exact phrases"</code> - normally we return results with similar phrases, but putting it inside quotes makes sure that we return only results that have the expression in that form somewhere.</li>
-    <li><code>package:prefix</code> - return only packages that begin with <code>prefix</code>. Useful to search for packages of the same framework.</li>
-    <li><code>dependency:package_name</code> - return only packages that reference <code>package_name</code> in their pubspec</li>
-    <li><code>dependency*:package_name</code> - return only packages that depend on <code>package_name</code> (direct, dev or transitive dependencies)</li>
+    <li><code>"exact phrase"</code>: By default, when you perform a search,
+    the results include packages with similar phrases. When a phrase is inside
+    quotes, you'll see only those packages that contain exactly the specified
+    phrase.</li>
+    <li><code>package:prefix</code>: Searches for packages that begin with
+    <code>prefix</code>. Use this feature to find packages in the same framework.
+    </li>
+    <li><code>dependency:package_name</code>: Searches for packages that reference
+    <code>package_name</code> in their pubspec.</li>
+    <li><code>dependency*:package_name</code>: Searches for packages that depend
+    on <code>package_name</code> (as direct, dev, or transitive dependencies).
+    </li>
   </ul>
   <h2 id="scoring">Scoring</h2>
   <p>
-    We calculate <a href="#popularity">popularity</a>, <a href="#health">health</a> and
-    <a href="#maintenance">maintenance</a> scores independently, and combine them into
-    an <a href="#overall-score">overall score</a>. The score you see in the colored circle
-    is always this overall score.
+    For each package, this site displays an <b>overall score</b>, calculated
+    from scores for the package's <a href="#popularity">popularity</a>,
+    <a href="#health">health</a>, and <a href="#maintenance">maintenance</a>.
   </p>
 
   <h3 id="popularity">Popularity</h3>
   <p>
-    Popularity score is good proxy for a package's use, derived from download statistics.
-    It is based on actual download counts, but compensated for automated tools (like
-    continuous builds fetching the package on each change request).
+    The popularity score—representing how often a package is used—is derived
```

Comment:
`used—is` - this looks strange

Reply:
I'll check the appearance once it's up... it might not be so odd once it's displayed using better fonts.